go.mod

```diff
@@ -8,7 +8,6 @@ require (
 	github.com/containerd/console v1.0.3
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/cyphar/filepath-securejoin v0.2.3
-	github.com/docker/go-units v0.4.0
```

Comment:
I pushed v0.5.0 so you could update to that instead if you want to

Reply:
Thanks, replace the docker/units removal with the bump commit; PTAL

Reply:
Just out of curiosity, I have re-applied the commit that replaces docker/units with the in-house implementation of RAMInBytes, and the difference between these two is negligible (meaning, I guess, that the linker is smart enough to remove the functions that are not used). Here are the raw numbers:

```console
[kir@kir-rhat runc]$ size runc-no-regex runc-no-units
   text	   data	    bss	    dec	    hex	filename
5522150	3598080	 229384	9349614	 8ea9ee	runc-no-regex
5520182	3597232	 229352	9346766	 8e9ece	runc-no-units
[kir@kir-rhat runc]$ ls -l  runc-no-regex runc-no-units
-rwxr-xr-x. 1 kir kir 9129056 Aug 31 18:26 runc-no-regex
-rwxr-xr-x. 1 kir kir 9128992 Aug 31 18:32 runc-no-units
```